src/Display/Progress/Progress.tsx

```diff
@@ -23,10 +23,12 @@ const Progress: React.FunctionComponent<Props> = (props) => {
 
   let normalizedPercentage;
   let normalizedSize;
-
-  if (percentage < 0 || percentage > 100) {
+  if (isNumber(percentage) && percentage < 0 || percentage > 100) {
```

Comment:
@franciscrispin this will evaluate to `true` even if isNumber(percentage) is false, and somehow percentage > 100 is `true`. You'll require 1 more set of brackets.